cpp/src/io/parquet/decode_fixed.cu

```diff
@@ -993,6 +1000,59 @@ CUDF_KERNEL void __launch_bounds__(decode_block_size_t, 8)
     return;
   }
 
+  // MH: We need something done for lists
```

Comment:
Some state other than string offsets must be updated for lists but not fully sure what.